deluge/core/torrentmanager.py

```diff
@@ -1392,7 +1392,22 @@ def on_alert_tracker_error(self, alert):
         log.debug(
             'Tracker Error Alert: %s [%s]', decode_bytes(alert.message()), error_message
         )
-        torrent.set_tracker_status('Error: ' + error_message)
+        if lt.version >= '1.2':
```

Comment:
This is comparing strings so the version comparison will not work as you expect it to. Look for other examples of version comparisons in the code.

Reply:
Fixed :)